### DISCUSSION TITLE
Allow loading of custom fonts from a URL

### DISCUSSION THREAD

**Discussion by thaddeuskkr** [monkeytypegame/monkeytype#4975] [Ideas]
the title says it all. it would be nice to be able to import a font from e.g. google fonts using a font URL (just like the `@import` function in CSS). it would make the QOL for people using monkeytype on multiple devices (like me) better as they wouldn't have to go through the process of installing the font on every device they use.

**Comment by realbakke** [comment]
yeah that would be very nice
